R/population_usage.R

```diff
@@ -0,0 +1,94 @@
+#' Estimate average population usage given at equilibrium
+#'
+#'
+#' @param distribution The % of the population who are distributed a net
+#' @param distribution_frequency The frequency of the distribution cycle (years)
+#' @param net_loss_hl The half life of the exponentially decaying net loss function (years)
+#'
+#' @return The equilibrium population-level net usage
+#' @export
+exp_loss_equilibrium <- function(distribution, distribution_frequency, net_loss_hl){
+  distribution * net_loss_hl / distribution_frequency * (1 - exp(-distribution_frequency / net_loss_hl))
+}
+
+#' Sample those to receive an intervention with correlation
+#' 
+#' Code adapted from malariasimulation
+#'
+#' @param p Probability of receiving a bed net
+#' @param mvnorm A vector of multivariate normal (correlated) draws.
+#' @param sigma_squared Sigma squared
+#' 
+#' @importFrom stats qnorm rnorm
+#'
+#' @return A boolean vector denoting which individuals will receive the intervention
+sample_intervention <- function(p, mvnorm, sigma_squared) {
+  sd <- sqrt(1 + sigma_squared)
+  u0 <- -qnorm(p, 0) * sd
```

Comment:
Question - do you not have to call the package when you use the importFrom command vs. the import command?
Or is it still best practice to do stats::qnorm
Asking because I dont know

Reply:
I actually prefer the `stats::qnorm` option as it is more explicit. Have changed